public/app/features/dashboard-scene/serialization/__snapshots__/transformSceneToSaveModelSchemaV2.test.ts.snap

```diff
@@ -2,7 +2,66 @@
 
 exports[`transformSceneToSaveModelSchemaV2 should transform scene to save model schema v2 1`] = `
 {
-  "annotations": [],
+  "annotations": [
+    {
+      "kind": "AnnotationQuery",
+      "spec": {
+        "datasource": {
```

Comment:
datasource can be undefined because we are making it optional in veneer here (https://github.com/grafana/grafana/blob/20d2bd934376853a33c605e6b5ddb55af4fa914c/packages/grafana-schema/src/veneer/dashboard.types.ts#L37).